alacritty/src/renderer/text/builtin_font.rs

```diff
@@ -660,7 +660,7 @@ fn powerline_drawing(
     })
 }
 
-#[repr(packed)]
+#[repr(Rust, packed)]
```

Comment:
isn't that repr `C` before? So to preserve we should keep it. Also, IIRC it's used in shaders, thus should be `C`.

Reply:
No, `#[repr(packed)` is Rust, not C. I'll change it to `C` just to make sure.